store/tikv/gcworker/gc_worker.go

```diff
@@ -490,6 +490,9 @@ func (w *GCWorker) calculateNewSafePoint(now time.Time) (*time.Time, error) {
 }
 
 func (w *GCWorker) runGCJob(ctx context.Context, safePoint uint64, concurrency int) error {
+	failpoint.Inject("mockRunGCJobFail", func() {
```

Comment:
Is this used for `automated-tests`, we could link the test in this change

Reply:
It's not used by automated-tests currently. I added it for release testing.